libs/blocks/aside/aside.css

```
@@ -370,19 +450,84 @@
   justify-content: flex-start;
 }
 
+.aside.split .foreground.container .text .icon-stack-area {
+  display: flex;
+  flex-flow: row wrap;
+  gap: 12px;
+  margin: -8px 0 var(--spacing-s);
+  list-style-type: none;
+  padding: 0;
+  width: 100%;
+  flex-direction: column;
```

```suggestion
  display: flex;
  flex-flow: row wrap;
  flex-direction: column;
  gap: 12px;
  margin: -8px 0 var(--spacing-s);
  width: 100%;
  padding: 0;
  list-style-type: none;
```

nit: grouping/readability

---

@fullcolorcoder I have made the requested changes.